hack/helm/trento-dev/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: trento-server
+  repository: file://../../../packaging/helm/trento-server
+  version: 0.1.0
+digest: sha256:ca1dd3126e0344280cff27f3cb8c67545c8679aaddc3644d9690154d2b7d59c2
```

Comment:
Does this mean that we will have to manually update this each time the trento-server helm chart gets updated? or does it happen automatically?

Reply:
correct